.github/workflows/build.yml

```diff
@@ -99,4 +99,5 @@ jobs:
           gh release create "${GITHUB_REF_NAME}" \
             --discussion-category Announcements \
             --notes-file release-note.md \
+            --repo groonga/docker \
```

Comment:
It might be better to use `${GITHUB_REPOSITORY}`.

Reply:
Oh... I see. I've fixed it.